index.html

```diff
@@ -155,8 +156,11 @@
       // maybe disconnect after processing the events.
       observer.disconnect();
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({ entryTypes: ["resource", "mark", "measure"] });
+    observer.observe({
```

Comment:
Let's keep this simple and stick to Resource Timing and User Timing.. those are uniformly supported by all browsers already.

```
// retrieve buffered events and subscribe to new events
// for Resource Timing and User Timing
observer.observe({
  entryTypes: ["resource", "mark", "measure"],
  buffered: true
});